tests/launchsecurity/sev.go

```diff
@@ -1,27 +1,180 @@
+		}
+
+		expectedSEVMeasurementInfo = v1.SEVMeasurementInfo{
+			APIMajor:  0,
+			APIMinor:  17,
+			BuildID:   46,
+			Policy:    1,
+			LoaderSHA: "a9da39a73edf4a7ba84e03a7e373c147c2d3df255051a6a130df4c2dd0d569d8",
+		}
+	)
+
+	newSEVFedora := func(opts ...libvmi.Option) *v1.VirtualMachineInstance {
+		const secureBoot = false
```

Comment:
nit: maybe rename to "withoutSecureBoot"? or simply `libvmi.WithUefi(false)`, which is clear enough IMO